code/game/turfs/space/space.dm

```diff
@@ -13,12 +12,11 @@
 	LAZYINITLIST(dust_cache)
 	for (var/i in 0 to 25)
 		var/image/im = image('icons/turf/space_dust.dmi',"[i]")
-		im.plane = DUST_PLANE
+		im.layer = DUST_LAYER
```

Comment:
I have a feeling that this and the SPACE_PLANE change may break parts of skybox and/or its parallax.

Reply:
Currently dust isn't showing up, I'm quite confused as to why.

Reply:
This is fixed.